src/FreeImageBmp.h

```diff
@@ -7,12 +7,21 @@
 class FreeImageBmp
 {
 public:
-	//Takes ownership of fiBitmap
-	//FreeImageBmp(FIBITMAP* fiBitmap);
+	FreeImageBmp(FreeImageBmp&& other); // Move constructor
+
+	// Create a new, default bitmap (pixels are all black)
 	FreeImageBmp(int width, int height, int bpp);
+
+	// Create bitmap from memory stream
 	FreeImageBmp(FREE_IMAGE_FORMAT imageType, FIMEMORY* fiMemory);
+
+	// Create a bitmap from a file
 	FreeImageBmp(FREE_IMAGE_FORMAT imageFormat, const std::string& filename);
+
+	// Create a new bitmap by scaling the size and width of an existing bitmap
+	FreeImageBmp(const FreeImageBmp& freeImageBmp, int scaledWidth, int scaledHeight);
```

Comment:
The scaling copy constructor is a really nice touch. (Though I suppose technically not a formal "copy constructor" since it takes parameters other than a class of the same type).